.bazelrc

```diff
@@ -20,7 +20,8 @@ build --copt -Wempty-body
 build --copt -Wenum-conversion
 build --copt -Wexpansion-to-defined
 build --copt -Wignored-qualifiers
-build --copt -Wimplicit-fallthrough=3
+# XXX: zlib
```

Comment:
can we only unset this in some spots?

Reply:
- https://stackoverflow.com/questions/56401584/compiling-external-dependencies-with-different-compiler-flags
- Could be neat to play around with pigweed: https://github.com/google/pigweed/blob/main/pw_build/pigweed.bzl